framework/console/controllers/MessageController.php

```diff
@@ -93,6 +96,9 @@ public function actionExtract($configFile)
             'format' => 'php',
             'ignoreCategories' => [],
         ], require($configFile));
+        if (!empty($specifiedLanguage)) {
+            $config['languages'] = [ $specifiedLanguage ];
```

Comment:
We can use comma-separated list of language's codes and then explode it in array:
```php
$config['languages'] = explode(",", $languages);
```